main.js

```diff
@@ -1,13 +1,35 @@
+function arrayCheckEquals(array1, array2) {
+    if ((!array1) || (!array2))
+        return false;
+
+    if (array1.length != array2.length)
```

Comment:
Either add in the `{}` or do ` if (array1.length != array2.length) return false;`